backend/app2/api/dependencies.py

```
@@ -0,0 +1,83 @@
+    
+def get_project_repository() -> ProjectRepository:
+    """Get the project repository instance"""
+    return ProjectRepository()
+    
+def get_sound_repository() -> SoundRepository:
+    """Get the sound repository instance"""
+    return SoundRepository()
+    
+# Service instances
+def get_auth_service(
+    user_repository: UserRepository = Depends(get_user_repository)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)